README.md

```diff
@@ -1,35 +1,23 @@
-# UltraViolet 💜
+# Ultraviolet 
+
 [![ci:test](https://github.com/nyudlts/ultraviolet/workflows/ci:test/badge.svg)](https://github.com/nyudlts/ultraviolet/actions)
 [![ci:test:docs](https://github.com/nyudlts/ultraviolet/workflows/ci:test:docs/badge.svg)](https://nyudlts.github.io/ultraviolet/)
 
-UltraViolet is NYU Libraries' platform for long-term access to a broad range of NYU-related digital content, including open access data associated with research and scholarly output and data licensed for use by members of the NYU community. This repository is the codebase for UltraViolet, a local deployment of the [InvenioRDM framework](https://inveniordm.docs.cern.ch/).
-
-## Getting Started
+UltraViolet is NYU Libraries' platform for long-term access to a broad range of NYU-related digital content, including open access data associated with research and scholarly output and data licensed for use by members of the NYU community.  
+UltraViolet is a local deployment of the [InvenioRDM framework](https://inveniordm.docs.cern.ch/) maintained by CERN.  
```

Comment:
In my understanding InvenioRDM is not a framework, it uses Invenio framework to build turn-key repository

Reply:
Ok, clarified language for this one too